Arena.toml

```diff
@@ -1194,3 +1549,63 @@ permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4
 document = "stjudecloud/workflows:/workflows/qc/quality-check-standard.wdl"
 message = "quality-check-standard.wdl:111:1: note[LineWidth]: line exceeds maximum width of 90"
 permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/workflows/qc/quality-check-standard.wdl/#L111"
+
+[[diagnostics]]
+document = "stjudecloud/workflows:/workflows/qc/quality-check-standard.wdl"
+message = "quality-check-standard.wdl:124:13: note[CommentWhitespace]: comment not sufficiently indented"
+permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/workflows/qc/quality-check-standard.wdl/#L124"
```

Comment:
This seems fine to me. What's triggering this one?

Reply:
```
note[CommentWhitespace]: comment not sufficiently indented
    ┌─ ../workflows/workflows/qc/quality-check-standard.wdl:124:13
    │
124 │             # 0x4 (unmapped) || 0x100 (secondary) || 0x800 (supplementary)
    │             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    │
    = fix: this comment has 3 levels of indentation. It should have 4 levels of indentation.
```

Reply:
This comment is a child of a `LiteralMapNode` since it is part of a `struct` declaration.

Reply:
I think you would agree the "fix" here is wrong:
```
        FlagFilter comparative_filter = {
            "include_if_all": "0x0",
                # 0x4 (unmapped) || 0x100 (secondary) || 0x800 (supplementary)
            "exclude_if_any": "0x904",
            "include_if_any": "0x0",
            "exclude_if_all": "0x0",
        }
```
What should we do?

Reply:
To me, the fix would actually be:
```
        FlagFilter comparative_filter = {
                "include_if_all": "0x0",
                # 0x4 (unmapped) || 0x100 (secondary) || 0x800 (supplementary)
                "exclude_if_any": "0x904",
                "include_if_any": "0x0",
                "exclude_if_all": "0x0",
        }
```

We can skip certain ancestors, but that's going to be more complicated. I suppose you could do something like checking each ancestor to see if it is separated from the prior ancestor by a newline. That might be sufficient.

Reply:
I have a strong dislike for changing indent levels in increments >1. So I don't like that first suggestion on principle lol. So I pointed this out as being problematic over Slack, but this declaration is actually using map->struct coercion which IMO should not be permitted. Does the "correct" implementation have the same issue?
```
     FlagFilter comparative_filter = FlagFilter {
            include_if_all: "0x0",
            # 0x4 (unmapped) || 0x100 (secondary) || 0x800 (supplementary)
            exclude_if_any: "0x904",
            include_if_any: "0x0",
            exclude_if_all: "0x0",
    }
```
Can you check?